docs/pages/docs/getting-started/add-to-project.mdx

```diff
@@ -102,15 +102,15 @@ This means that Turbo can schedule them separately.
   </Tab>
 </Tabs>
 
-4. **Try running `build` and `lint` with `turbo`:**
+3. **Try running `build` and `lint` with `turbo`:**
 
 ```bash
 turbo build lint
```

Comment:
are these commands going to work? without global install need prefix of package manager run command like `yarn turbo build`?

Reply:
Updating